src/rdo.rs

```diff
@@ -524,32 +520,31 @@ pub fn rdo_mode_decision(fi: &FrameInvariants, fs: &mut FrameState,
             !needs_rec
           );
 
-        let cost = wr.tell_frac() - tell;
-        let rd = if fi.use_tx_domain_distortion && !needs_rec {
-          compute_tx_rd_cost(
+        let rate = wr.tell_frac() - tell;
+        let distortion = if fi.use_tx_domain_distortion && !needs_rec {
+          compute_tx_distortion(
             fi,
             fs,
             w,
             h,
             is_chroma_block,
             bo,
-            cost,
             tx_dist,
             skip,
             false
           )
         } else {
-          compute_rd_cost(
+          compute_distortion(
             fi,
             fs,
             w,
             h,
             is_chroma_block,
             bo,
-            cost,
             false
           )
         };
+        let rd = compute_rd(fi, rate, distortion);
         if rd < best.rd {
```

Comment:
I wanted : best.rd --> best.rd_cost, but it would be out of the scope of this patch....